api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/ClaimTest.java

```diff
@@ -86,4 +92,58 @@ void shouldReturnTheSameIdIfOneIsSet() {
         //Then
         Assertions.assertThat(id).isEqualTo(claim.getId());
     }
+
+    @Test
+    void shouldUpdateClaimStatusAndTimestamp() {
+        LocalDateTime originalTimestamp = LocalDateTime.now().minusHours(1);
+        Claim claim = Claim.builder()
+                .claimStatus(ClaimStatus.NEW)
+                .claimStatusTimestamp(originalTimestamp)
+                .build();
+        LocalDateTime now = LocalDateTime.now();
+
+        claim.updateClaimStatus(ClaimStatus.ACTIVE);
+
+        Assertions.assertThat(claim.getClaimStatusTimestamp()).isAfterOrEqualTo(now);
```

Comment:
This should just be `assertThat`, don't need/want the `Assertions` part in front of it.

Reply:
Unfortunately, there are other assertions in this class that make `Assertions.` necessary.

Reply:
e.g. `assertThat(violations).hasSingleConstraintViolation`